.travis.yml

```diff
@@ -20,6 +20,7 @@ jobs:
         - (pycodestyle --max-line-length=120 art || exit 0)
         - (pylint --disable=C0330,C0415,E203,E1136 -rn art || exit 0)
         - py.test --pep8 -m pep8
+        - (mypy art || exit 0)
```

Comment:
How about keeping the PEP8 run at the end, to make it easier to find for contributors when it fails. The other runs don't let the run fail for now.